docs/development/private-keys/secure-boot.md

```diff
@@ -92,6 +93,43 @@ curl <key> | gpg --import
     -   **almalinux-secureboot-ca.cer** file is the CA public key we should include into shim. It also should be imported to signing node to perform actual binaries signing.
     -   **almalinux-secureboot.cer** is Signing public key that may be recommended to include into some RPM packages (but not necessary)
 
+### Configure Yubikey PIV
+
+```bash
+# does the system see the key
+ykman piv info
+```
+
+Generate PIV Pins (only if new yubikey)
+```bash
+PIVPIN=$(awk '/^\w{6,8}$/' /usr/share/dict/words | shuf -n1)
+PIVPUK=$(openssl rand -base64 10 | tr -d +/= | cut -c 1-8)
+echo "PIV PIN: $PIVPIN; PIV PUK: $PIVPUK"
+```
+
+**Make sure to securely store the PIN and PUK.  They are specific to the PIV component of the Yubikey.**
+
+Increase PIN/PUK retry limit (only if new yubikey)
+```bash
+ykman piv access set-retries 9 9 --pin 123456 --force
```

Comment:
Because if we accidentally nuke one of the keys somehow it will be a headache.  9 is still low enough IMO and provides a little more buffer for mistakes, especially during setup/testing.